src/i_sound.c

```diff
@@ -797,20 +695,18 @@ boolean I_InitMusic(void)
     // Fall back to module 0 device 0.
     midi_player = 0;
     midi_player_module = music_modules[0].module;
-    if (midi_player_module != &music_sdl_module)
-    {
-       midi_player_module->I_InitMusic(0);
-    }
+    midi_player_module->I_InitMusic(0);
```

Comment:
Will this preserve us the ability to e.g. have the Fluidsynth backend selected for music and it hot-swaps to another backend (formerly SDL2_Mixer) if the next tracks turns out to be e.g. in MP3 format?

Won't we need a menu entry to select the OpenAL backend then? Or is it just the fallback for non-MIDI music and the choice is between Fluidsynth (and its soundfonts) and OPL emulation?

Reply:
> Will this preserve us the ability to e.g. have the Fluidsynth backend selected for music and it hot-swaps to another backend (formerly SDL2_Mixer) if the next tracks turns out to be e.g. in MP3 format?

Yes, hot swap works.

> Won't we need a menu entry to select the OpenAL backend then?

No, it's just a fallback to Ogg/FLAC/MP3/OPUS, whatever `libsndfile` supports.